site/layouts/partials/navbar.html

```diff
@@ -23,100 +23,102 @@
-          {{ $href = .RelPermalink -}}
-        {{ else -}}
-          {{ $href = .URL | relLangURL -}}
-        {{ end -}}
-        {{ $isExternal := ne $baseURL.Host (urls.Parse .URL).Host -}}
-        <a {{/**/ -}}
-          class="nav-link {{- if $active }} active {{- end }}" {{/**/ -}}
-          href="{{ $href }}"
-          {{- if $isExternal }} target="_blank" rel="noopener" {{- end -}}
-        >
-            {{- .Pre -}}
+            {{ if $cover }} td-navbar-cover {{ end }}" data-bs-theme="dark">
+  <div class="container-fluid flex-column flex-md-row">
+    <a class="navbar-brand" href="{{ .Site.Home.RelPermalink }}">
+      <span class="navbar-brand__logo navbar-logo">
+        {{ if ne .Site.Params.ui.navbar_logo false }}
+          {{ with resources.Get "icons/logo.svg" }}
+            {{ (. | minify).Content | safeHTML }}
+          {{ end }}
+        {{ end }}
+      </span>
+    </a>
+    <div class="td-navbar-nav-scroll ms-md-auto" id="main_navbar">
+      <ul class="navbar-nav">
+        {{ $p := . }}
+        {{ $s := .Site }}
+        {{ range .Site.Menus.main }}
+        {{ if .HasChildren }}
+        <!-- Menu items with children -->
+        <div class="dropdown">
+          <a class="nav-link dropdown-toggle" href="#" role="button" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false">
+            {{ .Pre }}
             <span>{{ .Name }}</span>
-            {{- .Post -}}
-        </a>
-      </li>
-      {{ end -}}
-      {{ end -}}
-      {{ end -}}
-      {{ if .Site.Params.versions -}}
-      <li class="nav-item dropdown d-none d-lg-block">
-        {{ partial "navbar-version-selector.html" . -}}
-      </li>
-      {{ end -}}
-      {{ if (gt (len .Site.Home.Translations) 0) -}}
-      <li class="nav-item dropdown d-none d-lg-block">
-        {{ partial "navbar-lang-selector.html" . -}}
-      </li>
-      {{ end -}}
-      {{ if .Site.Params.ui.showLightDarkModeMenu -}}
-      <li class="td-light-dark-menu nav-item dropdown">
-        {{ partial "theme-toggler" . }}
-      </li>
-      {{ end -}}
-    </ul>
-  </div>
-  <div class="d-none d-lg-block">
-    {{ partial "search-input.html" . }}
+          </a>
+          <ul class="dropdown-menu">
+            {{ $children := .Children.ByWeight }}
+            {{ if eq .Identifier "releases" }}
+            {{ with (partial "releasePages.html" (dict "site" $s)).active }}
+            {{ range . }}
+            <a class="dropdown-item" href="{{ .RelPermalink }}">{{ .Params.LinkTitle | default .Params.Title }}</a>
+            {{ end }}
+            {{ else }}
+            {{ $children = where .Children.ByWeight "Identifier" "ne" "all-releases-page" }}
+            {{ end }}
+            {{ end }}
+            {{ range $children }}
+            <a class="dropdown-item" href="{{ .URL }}">{{ .Name }}</a>
+            {{ end }}
+          </ul>
+        </div>
+        {{ else }}
+        <!-- Menu items without children -->
```

Comment:
Add the logic for menu items without children. Other changes in this PR are mainly formatting.

Reply:
Can you remove all the formatting changes? It's a bit hard to see the actual differences.

Reply:
Hi @snazy , removed the formatting changes.